topdown/strings.go

```diff
@@ -423,6 +441,7 @@ func init() {
 	RegisterFunctionalBuiltin1(ast.Upper.Name, builtinUpper)
 	RegisterFunctionalBuiltin1(ast.Lower.Name, builtinLower)
 	RegisterFunctionalBuiltin2(ast.Split.Name, builtinSplit)
+	RegisterFunctionalBuiltin2(ast.Repeat.Name, builtinRepeat)
```

Comment:
`RegisterFunctionalBuiltin2` is deprecated. We should use [RegisterBuiltinFunc](https://github.com/open-policy-agent/opa/blob/main/topdown/builtins.go#L81) instead. Then update the `builtinRepeat` to use [BuiltinFunc](https://github.com/open-policy-agent/opa/blob/main/topdown/builtins.go#L58).